apps/are_form/behaviours/first-page.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
can these be separated out as separate behaviours

Reply:
also first-page for a behaviour is not very clear what the behaviour does.  I know you didn't call it that

Reply:
Removed start-date date behaviour from the class, and added the validation to fields/first-page.js instead of behaviours